pkg/pb/gracefulexit.proto

```diff
@@ -77,22 +78,22 @@ message StorageNodeMessage {
 message NotReady {}
 
 message TransferPiece {
-  bytes piece_id    = 1 [(gogoproto.customtype) = "PieceID", (gogoproto.nullable) = false]; // the current piece-id
+  bytes piece_id    = 1 [(gogoproto.customtype) = "PieceID", (gogoproto.nullable) = false]; // the original piece ID
   bytes private_key = 2 [(gogoproto.customtype) = "PiecePrivateKey", (gogoproto.nullable) = false];
+
   // addressed_order_limit contains the new piece id.
   metainfo.AddressedOrderLimit addressed_order_limit =3;              
 }
 
 message DeletePiece {
-  bytes piece_id = 1 [(gogoproto.customtype) = "PieceID", (gogoproto.nullable) = false];
+  bytes piece_id = 1 [(gogoproto.customtype) = "PieceID", (gogoproto.nullable) = false]; // the original piece ID
```

Comment:
Rename the field to `original_piece_id` rather than using comments for it.

Also, why did we send the `DeletePiece` messages?

Reply:
Updated